app/javascript/chat/content.jsx

```diff
@@ -80,13 +88,16 @@ export default class Content extends Component {
                 'M20 3h2v6h-2V5h-4V3h4zM4 3h4v2H4v4H2V3h2zm16 16v-4h2v6h-6v-2h4zM4 19h4v2H2v-6h2v4z',
               )}
         </button>
-        <Display resource={resource} />
+        <Display
+          resource={resource}
+          closeReportAbuseForm={closeReportAbuseForm}
+        />
       </div>
     );
   }
 }
 
-const Display = ({ resource }) => {
+function Display({ resource, closeReportAbuseForm }) {
```

Comment:
Just out of curiosity, did we update this to `function` because it needs to be hoisted? Or is it a style/preference thing?

Reply:
Just a preference. We don't have a hat and fa at rule for this. It might make sense to add one to avoid debates and let the robots manage it.